modules/exploits/multi/local/xorg_x11_suid_server_modulepath.rb

```diff
@@ -0,0 +1,217 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = GoodRanking
+  include Msf::Exploit::EXE
+  include Msf::Exploit::FileDropper
+  include Msf::Post::File
+  include Msf::Post::Linux::Priv
+  include Msf::Post::Linux::Kernel
+  include Msf::Post::Linux::System
+
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Xorg X11 Server SUID privilege escalation',
```

Comment:
The name should be changed, given that the other module shares the same name. The module name should also match the file name.

Suggest:

```suggestion
      'Name'           => 'Xorg X11 Server SUID modulepath Privilege Escalation',
```